.circleci/docker/Dockerfile

```diff
@@ -46,10 +46,14 @@ ENV TPUVM_MODE "${tpuvm}"
 # https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64
 RUN rm /etc/apt/sources.list.d/nvidia-ml.list
 
+# Rotate nvidia repo public key (last updated: 04/27/2022)
+# Unfortunately, nvidia/cuda image is shipped with invalid public key
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub
```

Comment:
Thanks! So we have to update this link whenever they change their public key, right? If so, where can we find the new updated link in the future?

Reply:
Yes, but unless we change our nvidia/cuda image in the future. I started from https://developer.nvidia.com/, and there came across  https://developer.nvidia.com/blog/updating-the-cuda-linux-gpg-repository-key/. They will make an official announcement in a release note in the dev forum.